content/release-engineering/codefresh/best-practices.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Codefresh \"Best Practices\""
+description: "Our tips for working with Codefresh"
+tags:
+- best-practices
+---
+
+# Managing Secrets
+
+Use `chamber` to manage secrets. Provision a dedicated IAM user with limited scope by using the  [terraform-aws-iam-chamber-user](https://github.com/cloudposse/terraform-aws-iam-chamber-user) terraform module. Review our guide on [managing secrets with CI/CD]({{< relref "secrets-management/cicd.md" >}}).
```

Comment:
`terraform-aws-iam-chamber-user` is part of root modules https://github.com/cloudposse/terraform-root-modules/blob/master/aws/chamber/user.tf

Reply:
We should mention about this, to prevent provision additional `user`